presentation/src/main/java/com/shifthackz/aisdv1/presentation/screen/settings/SettingsScreen.kt

```diff
@@ -577,48 +577,50 @@ private fun ContentSettingsState(
         }
         //endregion
 
-        //region LINKS & OTHERS
-        SettingsHeader(
-            modifier = headerModifier,
-            loading = state.loading,
-            text = LocalizationR.string.settings_header_info.asUiText(),
-        )
-        SettingsItem(
-            modifier = itemModifier,
-            loading = state.loading,
-            startIcon = Icons.Default.MonetizationOn,
-            text = LocalizationR.string.settings_item_donate.asUiText(),
-            onClick = { processIntent(SettingsIntent.Action.Donate) },
-        )
-        SettingsItem(
-            modifier = itemModifier,
-            loading = state.loading,
-            startIcon = Icons.Default.AllInclusive,
-            text = LocalizationR.string.settings_item_on_boarding.asUiText(),
-            onClick = { processIntent(SettingsIntent.Action.OnBoarding) },
-        )
-        SettingsItem(
-            modifier = itemModifier,
-            loading = state.loading,
-            startIcon = Icons.Default.Report,
-            text = LocalizationR.string.settings_item_report_problem.asUiText(),
-            onClick = { processIntent(SettingsIntent.Action.ReportProblem) },
-        )
-        SettingsItem(
-            modifier = itemModifier,
-            loading = state.loading,
-            startIcon = Icons.Default.Gavel,
-            text = LocalizationR.string.settings_item_policy.asUiText(),
-            onClick = { processIntent(SettingsIntent.LaunchUrl.OpenPolicy) },
-        )
-        SettingsItem(
-            modifier = itemModifier,
-            loading = state.loading,
-            startIcon = Icons.Default.Code,
-            text = LocalizationR.string.settings_item_source.asUiText(),
-            onClick = { processIntent(SettingsIntent.LaunchUrl.OpenSourceCode) },
-        )
-        //endregion
+        if(!state.onBoardingDemo){
```

Comment:
Why this check was removed ?

Reply:
It was not removed but added. Did this to hide `Info` region as it does not show to the user when onboarding. Updated onboarding UI to look like this:
![](https://github.com/user-attachments/assets/43d8301b-5035-4ca7-bfd5-6e3ff09e236b)